scale/storage/test/brokers/test_nfs_broker.py

```diff
@@ -1,391 +1,203 @@
+        self.broker.load_configuration({'type': NfsBroker().broker_type, 'nfs_path': 'host:/path'})
+
     @patch('storage.brokers.nfs_broker.os.makedirs')
     @patch('storage.brokers.nfs_broker.os.path.exists')
     @patch('storage.brokers.nfs_broker.os.chmod')
-    @patch('storage.brokers.nfs_broker.nfs_mount')
-    @patch('storage.brokers.nfs_broker.nfs_umount')
     @patch('storage.brokers.nfs_broker.shutil.copy')
-    def test_successfully(self, mock_copy, mock_umount, mock_mount, mock_chmod, mock_exists, mock_makedirs):
-        '''Tests calling NfsBroker.upload_files() successfully'''
+    def test_successfully(self, mock_copy, mock_chmod, mock_exists, mock_makedirs):
+        """Tests calling NfsBroker.upload_files() successfully"""
 
         def new_exists(path):
             return False
         mock_exists.side_effect = new_exists
 
-        mount = 'host:/dir'
-        upload_dir = os.path.join('the', 'upload', 'dir')
-        work_dir = os.path.join('the', 'work', 'dir')
-        work_dir2 = os.path.join(os.sep, 'the', 'work', 'dir')
-        file_1 = 'my_file.txt'
-        file_2 = 'my_file.json'
-        local_path_file_1 = os.path.join('my_dir_1', file_1)
-        local_path_file_2 = os.path.join('my_dir_2', file_2)
-        workspace_path_file_1 = os.path.join('my_wrk_dir_1', file_1)
-        workspace_path_file_2 = os.path.join('my_wrk_dir_2', file_2)
-        workspace_path_file_3 = os.path.join(os.sep, 'my_wrk_dir_2', file_2)
-        full_local_path_file_1 = os.path.join(upload_dir, local_path_file_1)
-        full_local_path_file_2 = os.path.join(upload_dir, local_path_file_2)
-        full_workspace_path_file_1 = os.path.join(work_dir, workspace_path_file_1)
-        full_workspace_path_file_2 = os.path.join(work_dir, workspace_path_file_2)
-        full_workspace_path_file_3 = os.path.join(work_dir2, workspace_path_file_2)
+        volume_path = os.path.join('the', 'volume', 'path')
+        file_name_1 = 'my_file.txt'
+        file_name_2 = 'my_file.json'
+        local_path_file_1 = os.path.join('my_dir_1', file_name_1)
+        local_path_file_2 = os.path.join('my_dir_2', file_name_2)
+        workspace_path_file_1 = os.path.join('my_wrk_dir_1', file_name_1)
+        workspace_path_file_2 = os.path.join('my_wrk_dir_2', file_name_2)
+        full_workspace_path_file_1 = os.path.join(volume_path, workspace_path_file_1)
+        full_workspace_path_file_2 = os.path.join(volume_path, workspace_path_file_2)
+        file_1 = ScaleFile()
+        file_1.file_path = workspace_path_file_1
+        file_2 = ScaleFile()
+        file_2.file_path = workspace_path_file_2
+        file_1_up = FileUpload(file_1, local_path_file_1)
+        file_2_up = FileUpload(file_2, local_path_file_2)
 
         # Call method to test
-        mountstats_data = '''16 36 0:3 / /proc rw,nosuid,nodev,noexec,relatime shared:5 - proc proc rw
+        mountstats_data = """16 36 0:3 / /proc rw,nosuid,nodev,noexec,relatime shared:5 - proc proc rw
```

Comment:
I'm not sure. This was originally Trevor's code. I think bbcp uses it to see the NFS mount locations. I'm thinking we'll probably have to rework the bbcp stuff anyway now that everything is in a Docker container.